sphinx_automodapi/tests/helpers.py

```diff
@@ -5,15 +5,12 @@
 import sys
 from copy import deepcopy
 
-from ..utils import SPHINX_LT_17
+from sphinx.cmd.build import build_main
+
 from . import cython_testpackage  # noqa
 
-__all__ = ['build_main', 'write_conf', 'run_sphinx_in_tmpdir']
+__all__ = ['write_conf', 'run_sphinx_in_tmpdir']
```

Comment:
Is that OK to remove this from the namespace without a warning? I suppose yes as explicit imports still work.

Reply:
I only added this in that namespace in my last PR